ZotBinsAPI.py

```diff
@@ -282,7 +289,94 @@ def get_obervation_stats():
         print(e)
         return str(e)
 
+@app.route('/mocked/observation/get', methods=['GET'])
+def mocked_get_observation():
+    """ 
+    An API endpoint that mocks observation GET requests.
```

Comment:
Curious on when you think endpoint documentation is necessary in code. In quick glance, not all endpoints have it. My personal preference is documentation in form of swagger UI, and limit in code documentation.

Reply:
I think endpoints should be well documented in the code. I think it's a hassle if the developer has to switch back and forth between a documentation UI and coding. 

Why not both? I will add something to the readme as well.

Reply:
@gracechoe What do you think?

I'm okay with either but I don't want to add comment bloat if it isn't the standard for your subgroup.

Reply:
ZBCE might try out documentation in swagger UI, and we're (ZBCE & Zotbins UCI) actually thinking of working together on the API stuff going forward. Let's not worry about it for this codebase for now. If we choose to stick with this repo then we should probably look into swagger.

Reply:
Okay, will leave it in for now.